src/request/derive-address/DeriveAddress.js

```diff
@@ -64,19 +64,19 @@ class DeriveAddress {
     } // constructor
 
     /**
-     * @param {string} [passphrase]
+     * @param {string} [password]
```

Comment:
Is this the notation for `{string | undefined} password`?

Reply:
Yes. The `[]` brackets mean that the parameter can be left out, thus making it `undefined`.